docs/reference/ml/anomaly-detection/apis/estimate-model-memory.asciidoc

```diff
@@ -0,0 +1,74 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ml-estimate-model-memory]]
+=== Estimate {anomaly-jobs} Model Memory API
+++++
+<titleabbrev>Estimate Model Memory</titleabbrev>
+++++
+
+Estimates the model memory an analysis config is likely to need given
+cardinality estimates for the fields it references.
+
+[[ml-estimate-model-memory-request]]
+==== {api-request-title}
+
+`POST _ml/anomaly_detectors/_estimate_model_memory`
+
+[[ml-estimate-model-memory-prereqs]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have `manage_ml` or
+`manage` cluster privileges to use this API. See
+<<security-privileges>>.
+
+[[ml-estimate-model-memory-desc]]
+==== {api-description-title}
+
+This API enables you to estimate the model memory and {anomaly-job}
```

Comment:
> This API enables you....

I think this sentence repeats what we've already said in the intro so unless we have more info to add we can omit this section.